ffi/dart/test/icu_test.dart

```diff
@@ -64,7 +64,7 @@ void main() {
     expect(
         ZonedDateTimeFormatter.withLength(locale, DateTimeLength.long)
             .formatIso(zonedDateTime.date, zonedDateTime.time, zonedDateTime.zone),
-        '15. Januar 2025, 14:32:12 MEZ');
+        '15. Januar 2025, 14:32:12 Mitteleuropäische Zeit');
```

Comment:
I plan to add FFI to let users opt into this behavior, but this behavior should not be the default. FFI doesn't have semantic skeletons yet.

Reply:
yeah I can revert the diplomat change, it was just the best place to demonstrate this